src/transport/tests/LoopbackTransportManager.h

```diff
@@ -27,26 +27,26 @@ class LoopbackTransportManager
 {
 public:
     /// Initialize the underlying layers.
-    CHIP_ERROR Init()
+    static CHIP_ERROR Init()
```

Comment:
This class was not meant to be static. If you need static methods, please separate them somewhere else. Do not use a class as a `namespace` to place everything static inside it.

Reply:
Ok, I will keep this in mind when switching to static in followup PR